src/styles/card.css

```diff
@@ -0,0 +1,96 @@
+.card {
+  font-family: Roboto, sans-serif;
+  display: flex;
+  width: 200px;
+  flex-direction: column;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 16px;
+}
+
+.card__image {
+  width: 160px;
+  height: 134px;
+  margin: 16px 3px 0;
+}
+
+.card__title {
+  margin: 40px auto 0;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  color: var(--mainTextColor);
+}
+
+.card__subtitle {
+  margin: 4px 0 0;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  color: var(--secondaryTextColor);
+}
+
+.card__raiting {
+  display: flex;
+  justify-content: space-between;
+  margin: 16px 0 0;
+}
+
+.card__reviews {
+  font-style: normal;
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  color: var(--mainTextColor);
+}
+
+.card__prices {
+  margin: 24px 0 0;
+  display: flex;
+  justify-content: space-between;
+}
+
+.card__price {
+  font-style: normal;
+  font-weight: 400;
+  font-size: 12px;
+  line-height: 18px;
+  color: var(--secondaryTextColor);
+}
+
+.card__value {
+  font-style: normal;
+  font-weight: 700;
+  font-size: 16px;
+  line-height: 18px;
+  color: var(--mainTextColor);
+}
+
+.card__link {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  text-align: center;
+  text-transform: uppercase;
+  margin: 16px 0 0;
+  width: 166px;
+  height: 40px;
+  background: #00acdc;
+  border-radius: 5px;
+  color: #fff;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 14px;
+  line-height: 16px;
+  text-decoration: none;
+  border: 1px solid transparent;
+}
+
+.card__link:hover {
```

Comment:
add transition effect on hover

Reply:
I have question.
Where i can take value for transition?
Maybe i missed it.
Thank you.

Reply:
It's not required yet, it's more as recommendation here. You will definitely learn about transition later. 
If you are still interested in - read [`this`](https://developer.mozilla.org/en-US/docs/Web/CSS/transition)